app/controllers/nonsipp/landorproperty/LandPropertyAddressManualController.scala

```diff
@@ -199,13 +199,19 @@ object LandPropertyAddressManualController {
       "landPropertyAddressManual.field5.error.max"
     )
 
+  private val postCodeFormErrors = InputFormErrors.postcode(
+    "landOrPropertyPostcodeLookup.postcode.error.required",
```

Comment:
Would it be worth replacing this with an empty string, like with fields 2 and 3 on lines 176 and 183? Or is there a particular reason why this is here but the others have been replaced with empty strings?

The same is true for "landPropertyAddressManual.field5.error.required" on line 197.

Reply:
Yes, you are right this is unused, could be removed. As the PR is already merged, I'm going to add this to my next change. Reason for field5 being there is that is was still in the content document and messages file (removing it from there to). For postcode - copy/paste :)

Reply:
Removed them here: https://github.com/hmrc/pension-scheme-return-frontend/pull/897/commits/4d5cc2ba59271483e0b3ca3e7299103ec642139d FYI @ccatosdevelopment